experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -106,12 +126,18 @@ def runNeuralNetwork(boundingBox, conceptNode):
         featuresValue = boundingBox.get_value(PredicateNode('features'))
         if featuresValue is None:
             logger.error('no features found, return FALSE')
-            return TruthValue(0.0, 1.0)
+            return TruthValue(0.0, 0.0)
```

Comment:
Have you checked whether certainty is returned back by PatternMatcher?

Reply:
It isn't. We have to assign truth  value to some atom in atomspace.

Reply:
Actually I am not sure yet.. I get different results if Inheritance links have truth value and they don't..